doc/dev/release.md

```diff
@@ -45,6 +45,24 @@ $ git config [--global] user.signingkey "$GPG_KEY_ID"
 $ git config [--global] user.email "$GPG_EMAIL"
 ```
 
+Also, make sure that you setup the git gpg config as follows.
+```bash
+ cat ~/.gnupg/gpg.conf
+default-key $GPG_KEY_ID
+```
+
+> **NOTE** If you will do a release from an OSX then, to be able to assign the tag you need to do:
```

Comment:
Why are these block quoted?

Reply:
It was just to bring a better layout. However, if you did not like it then, I will change.

Reply:
I'd rather see `**Note for OSX users:**. The block quote is too aggressive IMO.